test/stripe/stripe_object_test.rb

```diff
@@ -213,28 +213,28 @@ def to_hash
     should "mass assign values with #update_attributes" do
       obj = Stripe::StripeObject.construct_from(id: 1, name: "Stripe")
 
-      obj.update_attributes({ name: "STRIPE" }) # rubocop:disable Style/BracesAroundHashParameters
+      obj.update_attributes({ name: "STRIPE" })
```

Comment:
This rule [has been removed](https://github.com/rubocop-hq/rubocop/releases/tag/v0.80.0).

https://github.com/rubocop-hq/rubocop/issues/7641